python/pyspark/ml/clustering.py

```diff
@@ -201,7 +202,74 @@ def computeCost(self, dataset):
         """
         return self._call_java("computeCost", dataset)
 
+    @since("2.1.0")
+    def hasSummary(self):
+        """
+        Return true if there exists summary of model.
+        """
+        return self.summary is not None
```

Comment:
Good point. Other summaries have implemented this method by calling:

````python
return self._call_java("hasSummary")
````

About the unit test - how can you create a situation where the model does not have a summary?